docs/pages/deploying-vclusters/high-availability.mdx

```diff
@@ -2,13 +2,169 @@
 title: High Availability
 sidebar_label: High Availability
 ---
+By default, vCluster runs one instance of each of its components. That’s fine for many use cases, like ones that are very ephemeral (dev environments, CI/CD, etc.). But suppose your situation requires virtual clusters with more redundancy. In that case, you can use vCluster’s High Availability feature to run multiple copies of the vCluster components so that the cluster is more resistant to individual pods failing.
 
+# Prerequisites
+For this tutorial, you will need a multi-node Kubernetes cluster. This can be a local cluster, created with tools like Minikube or kind, or a remote cluster managed or hosted by a cloud provider. Or even a bare metal cluster if you are so inclined. You just need more than one node in the cluster.
```

Comment:
nit: 

This can be a local minikube or kind cluster,  or any full Kubernetes cluster. You just need more than one node in the cluster.